attestation_demo/README.txt

```diff
@@ -1,5 +1,5 @@
 ---------------------------
```

Comment:
This file should be updated to reflect the files that actually get built, and how to use them with pelz.

Reply:
Updated in [2b02c00](https://github.com/NationalSecurityAgency/pelz/pull/107/commits/2b02c00fb75bed6fea3a09063ae91ed448745314)